external/dhcpd/dhcpd_lwip.c

```diff
@@ -92,15 +92,42 @@
 #define DHCP_SERVER_PORT         67
 #define DHCP_CLIENT_PORT         68
 
-/****************************************************************************
- * Private Types
- ****************************************************************************/
-/****************************************************************************
- * Private Data
- ****************************************************************************/
+static pthread_t g_dhcpd_tid = -1;
+static int g_dhcpd_term = 0;
+
 /****************************************************************************
  * Private Functions
  ****************************************************************************/
+#define DHCPD_MQ_NAME "dhcpd_queue"
+
+void *_dhcpd_join_handler(void *arg)
+{
+	struct mq_attr attr;
+	attr.mq_maxmsg = 10;
+	attr.mq_msgsize = 10;
+	attr.mq_flags = 0;
+	attr.mq_curmsgs = 0;
+
+	mqd_t md = mq_open(DHCPD_MQ_NAME, O_RDWR | O_CREAT, 0666, &attr);
+	if (md == NULL) {
+		ndbg("Failed to open mq\n");
+		return NULL;
+	}
+
+	while (!g_dhcpd_term) {
+		char msg[2];
+		int msglen = 2;
+		int prio = 0;
+		int nbytes = mq_receive(md, msg, msglen, &prio);
+		if (nbytes <= 0) {
```

Comment:
Sender sends 't', receiver should check message is 't' or not..

Reply:
Applied. Thanks for the comment.